axelor-account/src/main/java/com/axelor/apps/account/service/ScaleServiceAccountImpl.java

```diff
@@ -0,0 +1,79 @@
+package com.axelor.apps.account.service;
+
+import com.axelor.apps.account.db.Invoice;
+import com.axelor.apps.account.db.InvoiceTerm;
+import com.axelor.apps.account.db.Move;
+import com.axelor.apps.account.db.MoveLine;
+import com.axelor.apps.base.db.Company;
+import com.axelor.apps.base.db.Currency;
+import com.axelor.apps.base.service.ScaleServiceImpl;
+import java.math.BigDecimal;
+
+public class ScaleServiceAccountImpl extends ScaleServiceImpl implements ScaleServiceAccount {
```

Comment:
please replace getScaledValue( [...], boolean isCompanyAmount) by two methods `getScaledValue()` and `getCompanyScaledValue()`

Reply:
Resolved